release-manager/custom-create-ocp-resources.sh

```diff
@@ -9,26 +9,29 @@ while [[ "$#" > 0 ]]; do case $1 in
   -c|--component) COMPONENT="$2"; shift;;
   -p=*|--project=*) PROJECT="${1#*=}";;
   -p|--project) PROJECT="$2"; shift;;
+  -n=*|--nexus-url=*) NEXUS_URL="${1#*=}";;
+  -n|--nexus-url) NEXUS_URL="$2"; shift;;
+  -r=*|--repo-base=*) REPO_BASE="${1#*=}";;
+  -r|--repo-base) REPO_BASE="$2"; shift;;
 
   *) echo "Unknown parameter passed: $1"; exit 1;;
 esac; shift; done
 
-jiraUrl=$(cat /data/ods-configuration/ods-project-quickstarters/boilerplates/release-manager/jenkins.env | grep JIRA_URL | awk -F "=" '{print $NF}')
-nexusUrl=$(cat /data/ods-configuration/ods-project-quickstarters/boilerplates/release-manager/jenkins.env | grep NEXUS_URL | awk -F "=" '{print $NF}')
-docgenUrl=$(cat /data/ods-configuration/ods-project-quickstarters/boilerplates/release-manager/jenkins.env | grep DOCGEN_URL | awk -F "=" '{print $NF}')
+jiraUrl=$(cat /data/ods-configuration/ods-core.env | grep JIRA_URL | awk -F "=" '{print $NF}')
+docgenUrl=$(cat /data/ods-configuration/ods-core.env | grep DOCGEN_URL | awk -F "=" '{print $NF}')
```

Comment:
my mistake. shouldnt this be $quickstarterBasepath/ods-config/...  ?

Reply:
@kiwo Oh yes, it should be. Are you ok with the other changes? I'd prefer to merge the PR, and then you can update the location?